### DISCUSSION TITLE
Configuring the "► Run | Debug" feature to work with Rust programs

### DISCUSSION THREAD

**Discussion by pieterdd** [LunarVim/LunarVim#3949] [Q&A] [answered]
As pointed out in [a related thread](https://github.com/LunarVim/LunarVim/discussions/2868), you can press `<Leader>ll` to bring up the Debug option. When I do so, I get this error:

```
Language server does not support command `rust-analyzer.debugSingle`. This command may require a client extension.
```

What would be the minimal amount of setup I'd need to do to make this work?

**Comment by LostInTheLogs** [comment, marked as answer]
take a look at https://github.com/LunarVim/starter.lvim/tree/rust-ide
